sdks/python/apache_beam/testing/analyzers/perf_analysis_test.py

```diff
@@ -218,9 +218,10 @@ def test_change_point_has_anomaly_marker_in_gh_description(self):
         line for line in description.split(constants._NEW_LINES_JOINER)
         if re.match(r'timestamp: .*, metric_value: .*', line.strip())),
                      '')
-    string_to_search = (
-        'timestamp: Wed Dec 31 19:00:15 1969, metric_value: 3.00 <---- Anomaly')
-    self.assertTrue(string_to_search in runs_info)
+    pattern = pattern = (
+        r'timestamp: .+ (\d{4}), metric_value: (\d+.\d+) <---- Anomaly')
```

Comment:
I think searching for specific anomaly value is fine, e.g. 'metric_value: 3.00 <---- Anomaly'. You can also modify the anomaly dataset slightly to be:

0, 0, 0, 0, 0, 3, 4, 4, 4 , then having `3 <--- Anomaly` in the test condition would also check  that anomaly is reported at the first occurrence of the elevated reading.

Reply:
FYI, test case runs would look like this

```
timestamp: Thu Jan  1 00:00:18 1970, metric_value: 4.00
timestamp: Thu Jan  1 00:00:17 1970, metric_value: 4.00
timestamp: Thu Jan  1 00:00:16 1970, metric_value: 4.00
timestamp: Thu Jan  1 00:00:15 1970, metric_value: 4.00
timestamp: Thu Jan  1 00:00:14 1970, metric_value: 4.00
timestamp: Thu Jan  1 00:00:13 1970, metric_value: 4.00
timestamp: Thu Jan  1 00:00:12 1970, metric_value: 3.00 <---- Anomaly
timestamp: Thu Jan  1 00:00:11 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:10 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:09 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:08 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:07 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:06 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:05 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:04 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:03 1970, metric_value: 0.00
timestamp: Thu Jan  1 00:00:02 1970, metric_value: 0.00
```